### PR TITLE
OSDOCS-10419#adding two new support regions

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Version:
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-10419
https://issues.redhat.com/browse/OSDOCS-10420

Link to docs preview:
https://75682--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-account.html#installation-gcp-regions_installing-gcp-account

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by brendan-daly-red-hat** [comment]
@yunjiang29, PTAL at this update.

**Comment by yunjiang29** [comment]
@jianli-wei PTAL, thanks

**Review by Srivaralakshmi — COMMENTED** [review]
@brendan-daly-red-hat I do not see the :_mod-docs-content-type: <TYPE> module file metadata for the module included in this PR. Suggest you fix this issue (throughout your component modules if it is a global case) by raising a separate JIRA for this and track it to closure.

File: modules/installation-gcp-regions.adoc [hunk 1]
```diff
@@ -10,6 +10,7 @@
 You can deploy an {product-title} cluster to the following Google Cloud Platform (GCP)
 regions:
 
+* `africa-south1` (Johannesburg, South Africa)
```
Comment (Srivaralakshmi) [inline modules/installation-gcp-regions.adoc:13]: @brendan-daly-red-hat Please refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

I trust you will fix this issue of missing module file metadata, so I am proceeding ahead with the merge without blocking this PR.

**Pull request merged**